lib/view/subject_progress_bar.dart

```diff
@@ -19,52 +19,99 @@ class SubjectProgressBar extends StatelessWidget {
               onTap: () => callback
                 ..dispatch(ChangeView(TimeTracking(subject: subject))),
               child: Card(
+                  margin:
+                      new EdgeInsets.symmetric(horizontal: 5.0, vertical: 2.0),
                   elevation: 5,
                   child: Container(
+                    margin: new EdgeInsets.symmetric(
+                        horizontal: 5.0, vertical: 5.0),
                     child: Column(
                       mainAxisSize: MainAxisSize.min,
                       mainAxisAlignment: MainAxisAlignment.spaceBetween,
                       verticalDirection: VerticalDirection.down,
                       children: <Widget>[
                         Row(
-                            mainAxisAlignment: MainAxisAlignment.center,
+                            mainAxisAlignment: MainAxisAlignment.spaceBetween,
                             children: <Widget>[
                               Text(
-                                subject.title,
+                                subject.title.toUpperCase(),
                                 style: TextStyle(
-                                    color: Colors.black, fontSize: 25),
+                                    color: Colors.black,
+                                    fontSize: 17,
+                                    fontWeight: FontWeight.bold),
                               ),
-                              Container(
-                                width: 20.0,
-                                height: 20.0,
-                              ),
-                              Container(
-                                  width: 20.0,
-                                  height: 20.0,
-                                  decoration: new BoxDecoration(
-                                    shape: BoxShape.circle,
-                                    color: subject.color,
```

Comment:
again : only my opinion: i would like to see the color point bigger ( no idea how much effort it costs but maybe take another shape like a rectangle.